lightning/src/ln/channel.rs

```diff
@@ -3445,13 +3449,22 @@ impl<SP: Deref> ChannelContext<SP> where SP::Target: SignerProvider  {
 		}
 	}
 
+	/// Returns funding_transaction unless it has been broadcast already
+	pub fn funding_transaction_unless_rebroadcast(&self) -> Option<Transaction> {
```

Comment:
Regarding the function name, will this also be `None` if the channel is not yet funded?

Reply:
Yes, if the channel is not yet funded, the funding transaction is `None`, it will return that. I make the description more clear.